### ISSUE TITLE
Stop button is not working

### ISSUE THREAD

**Issue by ARRY7686** [SandeepVashishtha/AlgoVisualizer#6]
**The stop button is not able to stop the sorting method**

**Comment by SandeepVashishtha** [comment]
@ARRY7686 , yes, I'm aware of it. I haven't had any free time lately, so if you could take care of this, I'd really appreciate it.

**Comment by ARRY7686** [comment]
@SandeepVashishtha Sure Assign it me
